app/controllers/phases_controller.rb

```diff
@@ -10,7 +10,9 @@ def index
   end
 
   def submissions
-    @submissions = @phase.submissions.includes(evaluator_submission_assignments: [:evaluator, :evaluation])
+    # This causes a SQL error when sorting by assignee_count -- refactor into the service??
+    # @submissions = @phase.submissions.includes(evaluator_submission_assignments: [:evaluator, :evaluation])
+    @submissions = @phase.submissions
```

Comment:
we should try to preserve the `includes` for the other filtered views, or resolve the SQL error for the assignee sort view. not sure the best way, but maybe we can apply the `includes` inside the SubmissionsSortAndFilterService since neither `set_submission_counts` or `set_submission_statuses` seem to rely on the included models. let's try that.